document/tsrc/revhist_desc.tex

```diff
@@ -0,0 +1,6 @@
+% SPDX-FileCopyrightText: 2025 IObundle
+%
+% SPDX-License-Identifier: MIT
+
+%Change the date according to your need
+\input{\NAME_version} & \today & JTS & Initial document version \\ \hline
```

Comment:
should we set an actual date instead of \today, which will update everytime that UG is built?